nni/experiment/launcher.py

```diff
@@ -97,7 +86,8 @@ def _start_rest_server(config: ExperimentConfig, port: int, debug: bool, experim
         from subprocess import CREATE_NEW_PROCESS_GROUP
         proc = Popen(cmd, cwd=node_dir, creationflags=CREATE_NEW_PROCESS_GROUP)
     else:
-        proc = Popen(cmd, cwd=node_dir)
+        import os
+        proc = Popen(cmd, cwd=node_dir, preexec_fn=os.setpgrp)
```

Comment:
why `preexec_fn` is not set in the previous version?

Reply:
Because it's not needed.
This is about terminating subprocess.